runway/model.py

```diff
@@ -74,6 +81,7 @@ def command_route(command_name):
                     deserialized_inputs[name] = inp.deserialize(value)
                 try:
                     results = command_fn(self.model, deserialized_inputs)
+                    self.millis_last_command = timestamp_millis()
```

Comment:
Thinking about it more, I'm wondering if getting the time since the last command _request_ (i.e. before passing the data to `command_fn`) might be more useful. If we use this metric to decide whether to shut down idle models, we don't want to accidentally stop models with pending long-running requests. What do you think?

Reply:
Ah that's a nice idea. I moved it up a line (before `results=command_fn()`) but then realized that our Flask server is single threaded (at least in the test environment) and so a long-running decorated command function will block calls to `GET /` even if we set the `millis_since_last_command` right before. This change is more useful for knowing how long a command took though, so I think its still a good change. What do you think?

Reply:
👍 Good point, we should also move to the server always being multi-threaded, dev and prod.

Reply:
Yeah sounds good. I made a ticket for that in the dev board. Not going to consider this a blocker then... You cool with this PR as-is then?